lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
But, if Alex Haskell files are shown as being Haskell and have the same syntax highlighter, it won't change anything from the current behaviour...